doc/code/qml_resource.rst

```diff
@@ -1,12 +1,12 @@
-qml.resources
+qml.resource
```

Comment:
Decided to go with `resource` in another PR which is not merged yet, that is why this change appears here.